src/services/class.service.ts

```diff
@@ -74,6 +69,15 @@ export class ClassService {
     }
   }
 
+  // Any data that hits this function is already known data so no checks needed
+  async addClass(classData: IRegisterData): Promise<string> {
+    await classData.classChan.createOverwrite(classData.user.id, {
```

Comment:
@tanndlin is this the right way to do perm overrides now?

Reply:
Yes, but why was this change necessary?

Reply:
Cause I couldn't get the this's to work, so I needed a function  that did not use this.